polkadot/node/overseer/src/lib.rs

```diff
@@ -60,6 +60,7 @@
 // unused dependencies can not work for test and examples at the same time
 // yielding false positives
 #![warn(missing_docs)]
+#![allow(dead_code)] // TODO https://github.com/paritytech/polkadot-sdk/issues/5793
```

Comment:
Consider suppressing dead code on a single struct rather than the whole module

Reply:
I tried on the `Overseer` struct, but it did not help. I guess it expands outside of that struct.